### ISSUE TITLE
Figure out why SKB mode does not work with some NICs

### ISSUE THREAD

**Issue by anwayde** [firedancer-io/firedancer#1680]
There are some intel NICs that fail with an error like:
```
WARNING 04-26 10:55:31.095717 121634 15   net:0 src/waltz/xdp/fd_xsk.c(846): xsk sendto failed xsk_fd=0 (6-ENXIO-no such device or address)
```

**Comment by mmcgee-jump** [comment]
Also need to add an operator error message if `sendto` fails after bind indicating the driver issue.

**Comment by ptaffet-jump** [comment]
Fixed by #1791 and #1809
